server.c

```diff
@@ -123,9 +123,22 @@ int main(void)
             if (status == 0) printf("Other connection closed");
             if (status == -1) perror("recv");
             printf("%s", str);
-            if (send(new_fd, "Hello, world!", 13, 0) == -1)
+
+	    // send file named index.html whenever we get a request
+	    FILE* home_pg = fopen("index.html", "r");
+	    fseek(home_pg, 0L, SEEK_END);
+	    int size = ftell(home_pg);
```

Comment:
let's proactively catch errors and return here as well. A return value of -1L indicates an error